freqtrade/strategy/interface.py

```diff
@@ -530,8 +530,8 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
                                                 current_time=date))
 
         if (ask_strategy.get('sell_profit_only', False)
-                and trade.calc_profit(rate=rate) <= ask_strategy.get('sell_profit_offset', 0)):
-            # Negative profits and sell_profit_only - ignore sell signal
+                and trade.calc_profit_ratio(rate=rate) <= ask_strategy.get('sell_profit_offset', 0)):
```

Comment:
a probably better way would be to use `current_profit` - which is the already calculated version of this.